source/particles/particle_handler.cc

```diff
@@ -1208,7 +1274,7 @@ namespace Particles
         if (cell->is_locally_owned())
           {
             std::set<unsigned int> cell_to_neighbor_subdomain;
-            for (const unsigned int v : cell->vertex_indices())
+            for (const unsigned int v : GeometryInfo<dim>::vertex_indices())
```

Comment:
I think some of your changes revert the changes made during the simplex workshop to make the `ParticleHandler` work on simplex meshes. Could you check these instances?